tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
The package is called `letsencrypt` on Ubuntu. And so is the binary for `certbot_script`.